package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "telemetry-events",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
The reason this is not a bug-fix, is because ES6 did not exist as a thing to be supported when this module was created, nor was it one of its goals. I don't understand what is gained by not doing 1.1.0 ?